CHANGELOG.md

```diff
@@ -0,0 +1,91 @@
+<!--
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+# OpenWhisk Core
+
+## Apache 1.0.0
+### Branch: [1.0.0](https://github.com/apache/openwhisk/tree/1.0.0)
+### Notable changes
```

Comment:
I reviewed all commits since 0.9.0 and arbitrarily chose notable changes.
Please let me know if anything is missing or should be removed.

Reply:
Thank you for doing this. That was quite a large list of changes to go through over the last couple years